app/Jobs/CleanupStaleMultiplexedConnections.php

```
@@ -216,7 +216,7 @@ private function removeMultiplexFile(string $muxFile, string $reason): void
         }
 
         $muxSocket = "/var/www/html/storage/app/ssh/mux/{$muxFile}";
-        $closeCommand = "ssh -O exit -o ControlPath={$muxSocket} localhost 2>/dev/null";
+        $closeCommand = "ssh -O stop -o ControlPath={$muxSocket} localhost 2>/dev/null";
         Process::run($closeCommand);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Process' in method 'removeMultiplexFile'. (undefined)

(StaticAccess)